src/app/store.ts

```diff
@@ -97,12 +107,28 @@ export function setUpStore(preloadedState?: Partial<RootState>) {
       maxAge: 1000,
     },
     middleware: (getDefaultMiddleware) => {
+      const production = process.env.NODE_ENV === "production";
```

Comment:
process shouldn't be available on the front end use `import.meta.env.PROD` 
https://vitejs.dev/guide/env-and-mode#env-variables